cpp/src/DataStorm/NodeSessionI.cpp

```diff
@@ -44,13 +43,10 @@ namespace
             try
             {
                 optional<SessionPrx> session;
-                updateNodeAndSessionProxy(publisher, session, current);
+                updateNodeAndSessionProxy(*publisher, session, current);
```

Comment:
Is it really valid/expected to get a null publisher proxy from time to time?

Reply:
We always pass nullopt to `updateNodeAndSessionProxy` when called from `initiateCreateSession`, at this point the `SessionPrx` is not yet available.